CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 4.5.1 (2021-02-05)
```

Comment:
```suggestion
# 4.5.1 (2021-02-08)
```
I'm assuming this will go out on Monday, so we should probably update to Monday's date.